lib/uploadcare/api/file_storage_api.rb

```diff
@@ -0,0 +1,29 @@
+module Uploadcare
+  module FileStorageApi
+    BATCH_SIZE = 100
+
+    def store_files(objects)
+      in_batches(objects) { |batch| put "/files/storage/", to_uuids(batch) }
+    end
+
+    def delete_files(objects)
+      in_batches(objects) { |batch| delete "/files/storage/", to_uuids(batch) }
+    end
+
+    private
+
+    def in_batches(enum)
+      enum.each_slice(BATCH_SIZE) { |batch| yield batch }
+    end
+
+    def to_uuids(objects)
+      objects.map do |object|
+        case object
+        when Uploadcare::Api::File then object.uuid
+        when /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/ then object
+        else raise(ArgumentError, "Unable to convert object to uuid: #{object}")
```

Comment:
raising here will abort the operation.
it's better to add a warning and ignore the object.